Orm/Xtensive.Orm.Tests/Issues/IssueGitHub0070_IncludeProviderParameterMismatchWithinParameterContext.cs

```diff
@@ -26,7 +26,7 @@ public Token(Session session, Guid id)
 namespace Xtensive.Orm.Tests.Issues
 {
   [TestFixture]
-  public class GitHubIssue_xxx_ParameterMismatch : AutoBuildTest
+  public class IssueGitHub0070_IncludeProviderParameterMismatchWithinParameterContext : AutoBuildTest
```

Comment:
Maybe it is better to say parameter conflict instead of mismatch?

Reply:
Yes, I agree.